provision.sh

```diff
@@ -27,6 +27,12 @@ wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
 echo "--------------- Installing both virtualenv and virtualenvwrapper ---"
 pip install virtualenvwrapper virtualenv
 
+# install postgresql and setup user
+echo "--------------- Installing postgresql ------------------------------"
+apt-get install -y python-dev libpq-dev postgresql postgresql-contrib
+su - postgres -c "createuser -s vagrant"
+su - vagrant -c "createdb $DB"
```

Comment:
@alayek I suggest we try to notify everyone on slack on what we are currently doing, especially use Trello more so that we don't do the same thing at the same time. This way we can divide our labour much better.